### DISCUSSION TITLE
Add short key to de-foucs or focus rustdesk

### DISCUSSION THREAD

**Discussion by zongtaowang** [rustdesk/rustdesk#7004] [General]
### What is the feature?
There have been discussions about focus, de-focus, gaining control, or losing control of RustDesk #6265 #1223 #4158 #4779. The current behavior appears to be that to focus RustDesk, I need to click my mouse within the RustDesk window. To de-focus RustDesk, I need to click my mouse on another application's window outside of RustDesk. However, it appears that  a straightforward solution is to define a special customizable shortcut key (possibly rarely used) to de-focus or focus RustDesk.

### Why do I want this feature? 

My use case is specific. I use a tablet to draw on the screen. Sometimes, I want to draw over the area occupied by RustDesk. For example, while reading something on a remote desktop through RustDesk and having ideas about the figures there, I want to draw on the local screen over the RustDesk remote window. Ideally, this should be achievable without quiting full-screen mode. Although my use case is specific, I believe this feature may benefit users in other scenarios.

**Comment by tanloong** [comment]
I totally agree. This will be really helpful. Virtualbox has the right-ctrl key as the default one to de-focus from it.

**Comment by tanloong** [comment]
I totally agree. This will be really helpful. Virtualbox has the right-ctrl key as the default key to de-focus from it.

**Comment by Arashm12** [comment]
I always use my remote sessions (VDIs, VMs, etc.) in full screen. In other tools, I can press Ctrl+Alt to defocus the remote session, then use Alt+Tab to switch apps on my local machine or Win+D to access the local desktop — which is incredibly useful.

I'm really surprised that something so small yet useful is missing, while there are a lot of complex features implemented.

**Comment by andrii-zhygalko** [comment]
This feature would be very useful
